### PR TITLE
Added parser for Atlassian Confluence access logs

### PR DESCRIPTION
**Pull request by Onager**
## One line description of pull request

Parser for Confluence access logs

## Description:

**Related issue (if applicable):** fixes #<plaso issue number here>

## Notes:
All contributions to Plaso undergo [code
review](https://github.com/log2timeline/l2tdocs/blob/main/process/Code%20review%20process.asciidoc). This makes sure
that the code has appropriate test coverage and conforms to the Plaso [style
guide](https://plaso.readthedocs.io/en/latest/sources/developer/Style-guide.html).

One of the maintainers will examine your code, and may request changes. Check off the items below in
order, and then a maintainer will review your code.

## Checklist:
  - [ ] Automated checks (Travis, Codecov, Codefactor )pass
  - [x] No new [new dependencies](https://plaso.readthedocs.io/en/latest/sources/developer/Adding-a-new-dependency.html) are required or l2tdevtools has been updated
  - [x] Reviewer assigned

### CODE REVIEW

**Comment by joachimmetz** [comment]
@Onager test are failing, could you PTAL

File: plaso/parsers/confluence_access.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class ConfluenceAccessEventData(events.EventData):
+  """Confluence access event data.
+
+  Attributes:
+    forwarded_for (str): request X-FORWARDED-FOR header value.
+    http_request_method (str): http request method.
```
Comment (joachimmetz) [inline plaso/parsers/confluence_access.py:24]: in doc string please rename 'http' to 'HTTP'

Comment (joachimmetz) [reply plaso/parsers/confluence_access.py:24]: also for URI and URL

File: plaso/parsers/confluence_access.py [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+      user_agent = event_data.http_request_user_agent.strip()
+      event_data.http_request_user_agent = user_agent
+
+    event_data.http_version = self._GetValueFromStructure(structure,
+                                                          'http_version')
+
+    parser_mediator.ProduceEventWithEventData(event, event_data)
+
+    # pylint: disable=unused-argument
+
```
Comment (joachimmetz) [inline plaso/parsers/confluence_access.py:287]: remove this empty line

**Review by joachimmetz — CHANGES_REQUESTED** [review]
Some small nits.

File: plaso/parsers/confluence_access.py (hunk 2 above)
Comment (Onager) [reply plaso/parsers/confluence_access.py]: Done

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**